forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -38,18 +45,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * A method to securely tokenize an EBT card via ForagePANEditText
+     * Tokenizes an EBT Card via a [ForagePANEditText][com.joinforage.forage.android.ui.ForagePANEditText] Element.
      *
-     * @param params The parameters required for tokenization, including
-     * reference to a ForagePANEditText view for card input.
+     * @param TokenizeEBTCardParams A model that passes a [`foragePanEditText`][com.joinforage.forage.android.ui.ForagePANEditText] instance, `customerId`, and `reusable` as the [TokenizeEBTCardParams] that Forage uses to tokenize an EBT Card.
+     * @return A [ForageApiResponse] object. 
      *
-     * @return A ForageAPIResponse indicating the success or failure of the operation.
-     * On success, returns a [PaymentMethod](https://docs.joinforage.app/reference/create-payment-method)
-     * token which can be securely stored and used for subsequent transactions. On failure,
-     * returns a detailed error response for proper handling.
+     * On success, the object includes a `ref` token that represents an instance of a Forage [`PaymentMethod`](https://docs.joinforage.app/reference/payment-methods#paymentmethod-object). You can store the token for future transactions, like to [`checkBalance`](checkBalance) or to [create a `Payment`](https://docs.joinforage.app/reference/create-a-payment) in Forage's database. 
      *
-     * @throws ForageConfigNotSetException If the passed ForagePANEditText instance
-     * hasn't had its ForageConfig set via .setForageConfig().
+     * On failure, the response includes a list of [ForageError][com.joinforage.forage.android.network.model.ForageError] objects that you can unpack to troubleshoot the issue.   
```

Comment:
Optional:

Maybe worth mentioning specific error code examples (in a KDoc-y way) like
[card_not_reusable](https://docs.joinforage.app/reference/errors#card_not_reusable) ?